dvc/repo/get.py

```diff
@@ -17,6 +20,19 @@
 logger = logging.getLogger(__name__)
 
 
+def _copy_git_file(repo, src, dst):
+    src_full_path = os.path.join(repo.root_dir, src)
+    dst_full_path = os.path.abspath(dst)
+
+    if repo.root_dir not in src_full_path:
+        raise FileOutsideRepoError(src)
+
+    if os.path.isdir(src_full_path):
+        shutil.copytree(src_full_path, dst_full_path)
```

Comment:
Is this feature supposed to work for remote git repos (i.e., those accessed through  git://, https://, ssh://)? Forgive me if I'm missing something obvious, but it seems to me that this will work only if the external repo is accessible through filesystem.